### DISCUSSION TITLE
Student developer pack,  student id upload problem

### DISCUSSION THREAD

**Discussion by janisadhi** [community/community#102747] [GitHub Education]
### Select Topic Area

Question

### Body

I have not hd webcam to click photo of id while processing application, is there any way to upload a image from a device

**Comment by KowyXYZ** [comment]
They still didnt fix that issue so quick answer no ... sorry i know its bad answer to this but a lot of ppl is having the same issue as you

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @janisadhi , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by Olayiwolaaa** [comment]
- **Use a Mobile Device:** If you have a smartphone or tablet with a camera, you can take a photo of your ID using the device's camera. After taking the photo, you can transfer it to your computer using a USB cable, email, or cloud storage services like Google Drive or Dropbox. Once transferred, you can upload the image from your computer to GitHub.

- **Use a Scanner:** If you have access to a scanner, you can scan your ID to create a digital image. After scanning the ID, you can transfer the image file to your computer using a USB drive or email and then upload it to GitHub.

- **Ask for Assistance:** If you don't have access to a webcam, smartphone, or scanner, consider asking a friend, family member, or colleague who has the necessary equipment to assist you. They can help you capture an image of your ID and transfer it to your computer for uploading to GitHub.

- **Contact GitHub Support:** If none of the above options are feasible for you, consider reaching out to GitHub Support for assistance. Explain your situation and inquire if there are alternative methods available for submitting your ID image as part of the application process.
